views/onboard.html.erb

```diff
@@ -0,0 +1,65 @@
+<% @title = "Patches" %>
```

Comment:
@NxtChg You probably forgot to update the title for that page to "Who's Onboard".